addons/l10n_ar/models/template_ar_base.py

```diff
@@ -13,6 +13,8 @@ def _get_ar_base_template_data(self):
             'property_account_payable_id': 'base_proveedores',
             'property_account_expense_categ_id': 'base_compra_mercaderia',
             'property_account_income_categ_id': 'base_venta_de_mercaderia',
```

Comment:
I think we no longer need to depend on category's accounts and change logic in `stock_account` to depend and set/get value from company's account